miio/gateway.py

```diff
@@ -755,6 +768,7 @@ class AqaraHT(SubDevice):
 
     accessor = "get_prop_sensor_ht"
     properties = ["temperature", "humidity", "pressure"]
+    _model = "lumi.weather.v1"
```

Comment:
It looks odd when the model is the only one with underscore in the beginning. Should this also be simply `model`?

Reply:
well the property getter (line 650) is already called model so that is why I included the underscore.
Besides underscore indicates it is an internall variable in python.
But I could also simply remove the property getter and remove the underscore to directly acces the variable....

Whatever you prefer...

Reply:
Ah, sorry, I skipped that part somehow.. No, this is good to go!